keras/random/seed_generator.py

```diff
@@ -89,6 +89,9 @@ def next(self, ordered=True):
             self.state.assign((seed_state + 1) * 5387 % 933199)
         return new_seed_value
 
+    def get_config(self):
```

Comment:
The object would also need a `from_config` classmethod since it does not inherit from an object that implements one.

Reply:
done